src/jit/gtlist.h

```diff
@@ -161,7 +161,7 @@ GTNODE(ADD_LO     , "+Lo"          ,1,GTK_BINOP)
 GTNODE(ADD_HI     , "+Hi"          ,1,GTK_BINOP)
 GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
-GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP)
+GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP) // A mul that returns the 2N bit result of an NxN multiply
```

Comment:
This is very confusing. As I understand it, most of these nodes represent a 32-bit result (the upper half), but this one represents a 64 bit result. And the MULHI above seems to be more the equivalent of these other XXX_HI ops. Shouldn't this be something different, e.g. MUL_LONG? or MUL_INT_LONG?